pgt-gov-bot/templates/deployment.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
     {{- include "pgt-gov-bot.labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.replicaCount }}
+  strategy:
+    type: Recreate
```

Comment:
Taking the opportunity to fix an issue when the deployment are updated, but the PVC volume is single-mounted, and avoid the new pod to start.